docs/content/docs/getting-started.md

```diff
@@ -125,21 +173,63 @@ Running in Docker
 
 The source should go to `main.go`.
 
-To run, please execute `docker-compose up`.
+To run, execute `docker-compose up`.
 
-A more detailed explanation of how it is running (and how to add live code reload) can be found in [*Go Docker dev environment* article](https://threedots.tech/post/go-docker-dev-environment-with-go-modules-and-live-code-reloading/).
+A more detailed explanation of how it is working (and how to add live code reload) can be found in [*Go Docker dev environment* article](https://threedots.tech/post/go-docker-dev-environment-with-go-modules-and-live-code-reloading/).
 {{% /collapse-box %}}
 {{< /collapse >}}
 
 {{% load-snippet-partial file="content/docs/getting-started/amqp/main.go" first_line_contains="package main" last_line_contains="process(messages)" %}}
 {{% load-snippet-partial file="content/docs/getting-started/amqp/main.go" first_line_contains="func process" %}}
 {{% /tabs-tab %}}
 
+{{% tabs-tab id="sql"%}}
+
+{{< collapse id="running_sql" >}}
+
+{{< collapse-toggle box_id="sql-docker" >}}
+Running in Docker
+{{% /collapse-toggle %}}
+{{% collapse-box id="sql-docker" %}}
+{{% load-snippet file="content/docs/getting-started/sql/docker-compose.yml" type="yaml" %}}
+
+The source should go to `main.go`.
+
+To run, execute `docker-compose up`.
+
+A more detailed explanation of how it is working (and how to add live code reload) can be found in [*Go Docker dev environment* article](https://threedots.tech/post/go-docker-dev-environment-with-go-modules-and-live-code-reloading/).
+{{% /collapse-box %}}
+{{< /collapse >}}
+
+{{% load-snippet-partial file="content/docs/getting-started/sql/main.go" first_line_contains="package main" last_line_contains="process(messages)" %}}
+{{% load-snippet-partial file="content/docs/getting-started/sql/main.go" first_line_contains="func process" %}}
+{{% /tabs-tab %}}
+
 {{% /tabs %}}
 
-### Publishing messages
+### Creating Messages
+
+Watermill doesn't enforce any message format. `NewMessage` expectes a slice of bytes as the payload. You can use
+strings, JSON, protobuf, Avro, gob or anything else that serializes to `[]byte`.
+
+The message UUID is optional, but recommended, as it helps with debugging.
 
-{{% tabs id="publishing" tabs="go-channel,kafka,nats-streaming,gcloud,amqp" labels="Go Channel,Kafka,NATS Streaming,Google Cloud Pub/Sub,RabbitMQ (AMQP)" %}}
+```go
+msg := message.NewMessage(watermill.NewUUID(), []byte("Hello, world!"))
+```
+
+### Publishing Messages
+
+`Publish` expects a topic and a `Message` to be published.
```

Comment:
`msg` is variadic, so rather:
```
	`Publish` expects a topic and one or more `Message`s to be published.
```

Reply:
btw, if it's variadic, then 0 messages may be passed.
I don't exactly know how the implementations handle an empty slice of messages, but I guess most just iterate over `[]` and `return nil`?

Reply:
Good point, added "one or more"  in 131b47695b6334095112ccef6d08d3f9e8ab741f

Most implementations will just iterate over the messages and don't do anything if the slice is empty. But I think it's not worth mentioning in the getting started, to keep it easy to grasp for new users. It's also an implementation detail that can vary between PubSubs, while the variadic `Message` is part of Watermill's interface.